### PR TITLE
Origin/brent/us 04

### PR DESCRIPTION
**Pull request by bentlee108**
implemented a slightly working gui

### CODE REVIEW

File: analysis/analysis.py [hunk 1]
```diff
@@ -24,11 +30,15 @@
 Returns:
     padded_sequences: the padded sequences to be analyzed by the model
 """
-def pre_process(tokenizer:Tokenizer, data, max_word_len):
+
```
Comment (hknehans) [inline analysis/analysis.py:33]: I'm not sure how I feel about 2 lines between the function's info and the function itself. I personally wouldn't go past 1 line. I think at the least, the lines between the previous code and the info should be greater than the lines between the info and the function if that makes sense. I'd like to hear what others' opinions are one it.

File: gui/main.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-
```
Comment (hknehans) [inline gui/main.py:1]: I've seen this done before in other projects where a file of just imports is created and then that file is imported to make the importing on these main files cleaner. Definitely not necessary, but what would you all think of that?

**Review by hknehans — APPROVED** [review]
Good work Brent! I didn't test out the functionality, but I believe you that it works. I just made a couple comments on little things that I'd like to hear what others think about them.

File: analysis/analysis.py (hunk 1 above)
Comment (bentlee108) [reply analysis/analysis.py:33]: I actually agree, but the code just kept re-formatting on save and I didn't feel like fixing it lol.

Comment (hknehans) [reply analysis/analysis.py:33]: Oh interesting. Maybe you have some additional plug-in doing that.
So should we go with one line between the info and the function? Or no extra line?
And then what about between the previous code and the info? If we go with no lines for the question above ^ , then I'm thinking just one line. If not, then two lines.

**Pull request merged**